examples/vic-gov-au/test/e2e/integration/smoke/DemoLandingPage.feature

```diff
@@ -37,6 +37,15 @@ Feature: Demo Page
     And the whats next component should exist
     And the share this component should exist
 
```

Comment:
I'd recommend not running this on the DemoLanding page. Firstly its not repeatable content - the content is variable on each install and will give you varying responses. Secondly its an artificially large page, I think we should benchmark by building a landing page fixture that reflects a real world large page.